lib/ansible/modules/network/panos/panos_security_policy.py

```diff
@@ -0,0 +1,516 @@
+            on information collected by GlobalProtect. For example, the user access level can be determined HIP that
+            notifies the firewall about the user's local configuration.
+        required: false
+        default: "any"
+    destination:
+        description:
+            - List of destination addresses.
+        required: false
+        default: "any"
+    application:
+        description:
+            - List of applications.
+        required: false
+        default: "any"
+    service:
+        description:
+            - List of services.
+        required: false
+        default: "application-default"
+    log_start:
+        description:
+            - Whether to log at session start.
+        required: false
+        default: false
+    log_end:
+        description:
+            - Whether to log at session end.
+        required: false
+        default: true
+    action:
+        description:
+            - Action to apply once rules maches.
+        required: false
+        default: "allow"
+    group_profile:
+        description: >
+            security profile group that is already defined in the system. This property supersedes antivirus,
+            vulnerability, spyware, url_filtering, file_blocking, data_filtering, and wildfire_analysis properties.
+        required: false
+        default: None
+    antivirus:
+        description:
+            - Name of the already defined antivirus profile.
+        required: false
+        default: None
+    vulnerability:
+        description:
+            - Name of the already defined vulnerability profile.
+        required: false
+        default: None
+    spyware:
+        description:
+            - Name of the already defined spyware profile.
+        required: false
+        default: None
+    url_filtering:
+        description:
+            - Name of the already defined url_filtering profile.
+        required: false
+        default: None
+    file_blocking:
+        description:
+            - Name of the already defined file_blocking profile.
+        required: false
+        default: None
+    data_filtering:
+        description:
+            - Name of the already defined data_filtering profile.
+        required: false
+        default: None
+    wildfire_analysis:
+        description:
+            - Name of the already defined wildfire_analysis profile.
+        required: false
+        default: None
+    devicegroup:
+        description: >
+            Device groups are used for the Panorama interaction with Firewall(s). The group must exists on Panorama.
+            If device group is not define we assume that we are contacting Firewall.
+        required: false
+        default: None
+    commit:
+        description:
+            - commit if changed
+        required: false
+        default: true
+'''
+
+EXAMPLES = '''
+# permit ssh to 1.1.1.1
```

Comment:
Examples should specify a `- name:` rather than a comment, please see http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#examples-block for details.